ui/components/multichain/app-header/app-header-unlocked-content.tsx

```diff
@@ -215,30 +212,63 @@ export const AppHeaderUnlockedContent = ({
     );
   }, [history, selectedMultichainAccountId]);
 
-  const networksSubtitle = useMemo(() => {
-    if (!isMultichainAccountsState2Enabled) {
-      return null;
-    }
+  const multichainAccountAppContent = useMemo(() => {
+    const networksLabel =
+      numberOfAccountsInGroup === 1 ? t('network') : t('networks');
 
-    const label = numberOfAccountsInGroup === 1 ? t('network') : t('networks');
     return (
-      <Text
-        color={TextColor.primaryDefault}
-        variant={TextVariant.bodyXs}
-        onClick={handleNetworksClick}
-        data-testid="networks-subtitle-test-id"
-        className="networks-subtitle"
-      >
-        {`${numberOfAccountsInGroup} ${label.toLowerCase()}`}
-      </Text>
+      <Box>
+        <Text
+          as="div"
+          display={Display.Flex}
+          flexDirection={FlexDirection.Column}
+          alignItems={AlignItems.flexStart}
+          ellipsis
+        >
+          <AccountPicker
+            address={''} // No address shown in multichain mode
+            name={selectedMultichainAccount.metadata.name}
+            showAvatarAccount={false}
+            onClick={() => {
+              history.push(ACCOUNT_LIST_PAGE_ROUTE);
+              trackEvent({
+                event: MetaMetricsEventName.NavAccountMenuOpened,
+                category: MetaMetricsEventCategory.Navigation,
+                properties: {
+                  location: 'Home',
+                },
+              });
+            }}
+            disabled={disableAccountPicker}
+            paddingLeft={0}
+            paddingRight={0}
+          />
+          <>{!isMultichainAccountsState2Enabled && CopyButton}</>
+        </Text>
+        <Text
+          color={TextColor.primaryDefault}
+          variant={TextVariant.bodyXs}
+          onClick={handleNetworksClick}
+          data-testid="networks-subtitle-test-id"
+          className="networks-subtitle"
+        >
+          {`${numberOfAccountsInGroup} ${networksLabel.toLowerCase()}`}
+        </Text>
+      </Box>
     );
   }, [
+    CopyButton,
+    disableAccountPicker,
     handleNetworksClick,
+    history,
     isMultichainAccountsState2Enabled,
     numberOfAccountsInGroup,
+    selectedMultichainAccount.metadata.name,
     t,
+    trackEvent,
   ]);
 
+  // TODO: [BIP-44] Delete this method once multichain accounts is released
```

Comment:
Can we add a ticket number here?

Reply:
Hey @gantunesr i didn't find any documents but it is usually a good practice to follow so we don't lose track of it. Also a designated ticket can hold more context and fix information for whoever picks it up in the future.

Reply:
I don't think we have any "public ticket link" for this though (if you were referring to an actual GH ticket/issue)

Reply:
I don't think external contributor have the access to jira ticket anyway, or you can add something like 
`JIRA-PROJECT_NAME-Ticketid`

Reply:
Done in https://github.com/MetaMask/metamask-extension/pull/35932/commits/bdaa95434ec494acb0e590366513831bb2e47dc9

I strongly think this should be part of our contributor docs to avoid this back and forward in the future since it can be easily referenced to and it can provide clarity on the pattern to be used